adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -730,25 +773,27 @@ def _perceive_ground_relations(self):
                             ):
                                 object_is_on_something = True
                     if not object_is_on_something:
-                        self._relation_perceptions.append(
+                        ground_relations.append(
                             on(
                                 self._objects_to_perceptions[situation_object],
                                 perceived_ground,
-                            )
+                            )[0]
```

Comment:
Why are you only appending the first relation returned by the `on` function? The relationship for the `contacts` is symmetric on purpose.

Reply:
When I was running the tests all I got from `on()` was a tuple with the relation I wanted on the inside and nothing else. @LichMaster98

Reply:
Looking at `on()` since we aren't passing `iterable`s (as far as I can tell) we should always get a `tuple` of size 1 with the relation we want.

Reply:
`on` returns a tuple but you should want to append both i.e:

`on(ground, book)` -> `(Relation(IN-REGION, ground, Region(book, distance=EXTERIOR_BUT_IN_CONTACT)), Relation(IN-REGION, book, Region(ground, distance=EXTERIOR_BUT_IN_CONTACT))` 

The program may return in the opposite order I don't recall but both of these relationships are true and should exist in the final output.

Reply:
When I was reviewing the error I found that the Tuple only had one relation each time. Following:
https://github.com/isi-vista/adam/blob/82e8089a4d2d9f1cd31178217b18854e22576deb/adam/ontology/phase1_ontology.py#L574
I found:
https://github.com/isi-vista/adam/blob/82e8089a4d2d9f1cd31178217b18854e22576deb/adam/relation.py#L245-L261
which by my understanding only returns a tuple larger than 1 if an actual iterable is used for the arguments which we don't use here. Let me know if there is anything I'm not seeing or if this changed somewhere that I'm again not seeing.

Reply:
@jamart28 Apologies, on isn't symmetric I'm just crazy. Regardless you shouldn't need to explicitly select 0. I think you may want to `extend` rather than `append`

Reply:
I can do that.